modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayProjectsAdministrator.testcase

```diff
@@ -0,0 +1,170 @@
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "Site Initializer Testray";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		//Testray.createTestraySite(siteName = "Testray 2");
+
+		Refresh();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		//Testray.tearDownCP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "This is a test for LPS-160539. Verify if as an Administrator I am able to add a Project"
+	@priority = "5"
+	test AdminCanAddProject {
+		task ("Go to Testray and Create Project") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			TestrayManage.addProjectManageBar(
+				description = "Projects_1 Project Description",
+				name = "Projects_1 Project");
+		}
+
+		task ("Check if the project was created") {
+			Testray.assertProjectName(key_name = "Projects_1 Project");
+
+			Testray.assertProjectDescription(value = "Projects_1 Project Description");
+		}
+
+		task ("TearDown") {
+			Testray.deleteProject(valueName = "Projects_1 Project");
+		}
+	}
+
+	@description = "This is a test for LPS-160538. Verify if as an Administrator I am able to delete a Project"
+	@priority = "5"
+	test AdminCanDeleteProject {
+		task ("Go to Testray and Create Project") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			Testray.createProjectTestray(
+				description = "Projects_2 Description",
+				key_name = "Projects_2 Project");
+		}
+
+		task ("Delete the project and make sure it is deleted") {
+			TestrayContextMenu.clickContextMenuProjects(projectName = "Projects_2 Project");
+
+			TestrayContextMenu.selectItemFromProjectsContextMenu(key_menuItem = "Delete");
+
+			Testray.assertProjectNotPresent(key_name = "Projects_2 Project");
+		}
+	}
+
+	@description = "This is a test for LPS-160537. Verify if as an Administrator I am able to edit a Project name and description"
+	@priority = "5"
+	test AdminCanEditProject {
+		task ("Go to Testray and Create Project") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			Testray.createProjectTestray(
+				description = "Projects_3 Description",
+				key_name = "Projects_3 Project");
+		}
+
+		task ("Edit Project name and description") {
+			TestrayContextMenu.clickContextMenuProjects(name = "Projects_3 Project");
+
+			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Edit");
+
+			Testray.editProjectTestray(
+				description = "Projects_3 Project Description Edited",
+				name = "Projects_3 Project Edited");
+		}
+
+		task ("Check if the name and description has been edited") {
+			TestrayLayout.clickResultsOnSidebar();
+
+			Testray.assertProjectName(key_name = "Projects_3 Project Edited");
+
+			Testray.assertProjectDescription(value = "Projects_3 Project Description Edited");
+		}
+
+		task ("TearDown") {
```

Comment:
@thaiscabrals I noticed a lot of tests with this task "TearDown". Please create a generic macro for deleting the projects, and add it to the tearDown() block of the .testcase